linux-pull-run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+open_browser=true
+while getopts 'o' OPTION; do
+    case "$OPTION" in
+        o)
+            open_browser=false
+            ;;
+        ?)
+            echo "script usage: ./linux-pull-run.sh [-o]" >&2
+            exit 1
+            ;;
+    esac
+done
+docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
+if [ "$open_browser" = true ] ; then
```

Comment:
```suggestion
while getopts 'o' OPTION; do
    case "$OPTION" in
        o)
            no_open=true
            ;;
        ?)
            echo "script usage: ./linux-pull-run.sh [-o]" >&2
            exit 1
            ;;
    esac
done
docker pull ghcr.io/badgerloop-software/chase-car-dashboard-image:latest
if [[ ! $no_open ]]; then
```
Works either way, but could simplify the condition for not opening the page

Reply:
Also, would be best to add descriptions of each flag to the usage message.